Contribution guidelines (#3)
redhat-documentation/publishing-tutorial-sample-repo: README.md

1. Ensure you are on main

    ```sh
    $ git checkout main
    ```
1. Ensure your fork is up to date

    ```sh
    $ git pull upstream main
    ```
1. Create a working branch based on the issue ID. For example for issue #2, you can use "2" or "02" for the branch name:

    ```sh
    $ git checkout -b 02
    ```       

### Creating a pull request and requesting a review

When your work is ready to be reviewed and merged, create a pull request.

1. Push your working branch to your fork:

    ```sh
    $ git push -u origin <branch_name>
    ```
1. From the repository page in GitHub, click **New pull request**.
1. Select your working branch from the compare list.
1. Add the `WIP:` prefix to the title of the pull request. This automatically converts the pull request to a draft pull request, preventing it from being merged.
1. Click **Create new pull request**.
1. Add the **awaiting tech review** label to the pull request.
1. In the pull request comment field, enter `@redhat-documentation/publishing-tutorial-team Please review for technical completeness and accuracy`.

### The review process

> At least one person on the project team reviews the merge request and adds comments in GitHub. This review process will be open for one week from the day the merge request was submitted. If the merge request is still being actively discussed beyond the one week timeframe, then the merge request will stay open. Once the merge request discussion is resolved, the merge request will be NACK'd or ACK'd based on the comments given. If no comments are given after a week, then a project administrator will ACK the merge request.

Reviewer: Hey @stoobie the PR looks good. So do I understand correctly that as a reviewer I don't need to do anything? That if I don't object it's going to be merged in a week?

# Publishing tutorial sample repo

> A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2"

Reviewer: ```suggestion
A sample repository that you can use during the "Getting started publishing with Pantheon 2" tutorial.
```

Author: Done.

* [Contributing to this repository](#contributing-to-this-repository)

## Contributing to this repository

### Filing a bug

> If you have any suggestions to improve or extend this repository, add a new issue in the repository.

Reviewer: ```suggestion
If you have any suggestions to improve or extend this repository, open a [new issue](https://github.com/redhat-documentation/publishing-tutorial-sample-repo/issues) in the repository.
```
I'd add the link because many teams don't track issues in their repos.

Author: Done.

### Prerequisites

> * You must have an account on GitHub.

Reviewer: Do we need to say "you must"? 
* Have a GitHub account.
* Have your SSH key added to your GitHub account.

Author: Fixed.

1. Click your user space in GitHub.

### Cloning the repository

After you have forked the repository, you must clone it to your local machine and add the original `redhat-documentation/publishing-tutorial-sample-repo` repository as an upstream remote.

1. From a terminal, clone the repository:

    ```sh
    $ git clone git@github.com:<user-space>/publishing-tutorial-sample-repo.git
    ```
1. Set up `redhat-documentation/publishing-tutorial-sample-repo` as the upstream:

    ```sh
    $ cd publishing-tutorial-sample-repo
    $ git remote add upstream git@github.com:redhat-documentation/publishing-tutorial-sample-repo.git
    ```

### Creating a working branch

Pull requests should have a corresponing issue in the repo. Whenever you work on a new issue, you must create a new working branch based on the latest version of the upstream main branch.

1. Ensure you are on main

    ```sh
    $ git checkout main
    ```
1. Ensure your fork is up to date

    ```sh
    $ git pull upstream main
    ```
1. Create a working branch based on the issue ID. For example for issue #2, you can use "2" or "02" for the branch name:

    ```sh
    $ git checkout -b 02
    ```       

> ### Creating a pull request and completing review

Reviewer: What do you mean by completing review? My suggestion would be "Creating a pull request and requesting a review" or "Submitting a pull request for review"

Author: Fixed.

When your work is ready to be reviewed and merged, create a pull request.

1. Push your working branch to your fork:

    ```sh
    $ git push -u origin <branch_name>
    ```
1. From the repository page in GitHub, click **New pull request**.
1. Select your working branch from the compare list.

> 1. Add `WIP:` to the title of the pull request. This automatically converts the pull request to a draft pull request, preventing it from being merged.

Reviewer: ```suggestion
1. Add `WIP:` prefix to the title of the pull request. This automatically converts the pull request to a draft pull request, preventing it from being merged.
```

Author: Done.

#### Documentation review
After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review complete**.

When technical review is complete, writers click the **Reviewers** gear icon and select the name of a team member to request peer review. The peer writer reviews the pull request for clarity, consistency, and compliance with necessary standards.
Writers receive a notification that comments have been added via email, and when all comments have been addressed, the reviewer clicks **Review changes > Approve** from the **Files changed** tab of the pull request to approve the changes and the pull request.

### Merging a pull request

When you have addressed all technical review and peer review comments, notify the developers to accept the pull request.

1. Remove `WIP` from the title of the pull request.
1. Click **Request Review** and enter `@@redhat-documentation/publishing-tutorial-team`.

> The developers check that the **Tech review complete** label has been added to the pull request and peer pull request approval provided, then accept it.

Reviewer: I'm a bit confused with "peer pull request approval provided" It seems that you wanted to say something like "that the reviewer approved the pull request" or something similar.

Author: Yeah, actually, I don't think we need this same process here. I changed this section a bit.
